### PR TITLE
HDB6 & HDB7:  doctors and patients pages

### PR DESCRIPTION
**Pull request by assawalhy**

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 			"something-went-wrong": "Something Went wrong"
 		}
 	},
-	"AddPatient": {
+	"Forms": {
 		"add-new-patient": "Add new patient",
```
Comment (mohamed-ahmed-yousef) [inline messages/en.json:11]: why change this to `Forms`

File: package.json [hunk 2]
```diff
@@ -27,13 +27,15 @@
 		"@mantine/tiptap": "^7.9.1",
 		"@tabler/icons-react": "^3.3.0",
 		"@tanstack/react-query": "^5.35.5",
+		"@tanstack/react-query-devtools": "^5.36.0",
```
Comment (mohamed-ahmed-yousef) [inline package.json:30]: I think we can move the react query dev tools to`devDependencies`

**Review by mohamed-ahmed-yousef — COMMENTED** [review]
At all LGTM,
- But we need to complete the `transitions` to all pages.
 - Another thing, I tried to login in through Netlify with conditionals `admin`  `admin` but it didn't work and it works in `postman`

File: messages/en.json (hunk 1 above)
Comment (assawalhy) [reply messages/en.json:11]: Will revert the change, inshallah.

**Review by mohamed-ahmed-yousef — APPROVED** [review]

**Pull request merged**